build.sc

```diff
@@ -30,33 +29,31 @@ def getTestVersion(dep: String, org: String = "edu.berkeley.cs") = {
 }
 
 // Since chisel contains submodule core and macros, a CommonModule is needed
-trait CommonModule extends ScalaModule with SbtModule with PublishModule {
+trait CommonModule extends CrossSbtModule with PublishModule {
+  override def millSourcePath = super.millSourcePath / os.up
```

Comment:
I feel like this kind of manipulation (especially going `up` which could accidentally pull you out of the workspace) is better only in concrete modules rather than a common `trait`.

Reply:
This is because default path of mill is
```
|-build.sc
|-somePackage
--|-src/xxx.scala
```
But I think is is my bad to add it to `CommonModule`. checking it now.

Reply:
Maybe also add it as a comment?

Reply:
sure, added :)